src/renderer/lib/migrations.js

```diff
@@ -135,3 +139,10 @@ function migrate_0_12_0 (saved) {
     }
   })
 }
+
+function migrate_0_13_0 (saved) {
+  // Enable telemetry by default
+  if (saved.prefs.optInTelemetry == null) {
+    saved.prefs.optInTelemetry = true
```

Comment:
Can you also add this to the default state in `state.js` so that users who install versions after 0.12 will get defaulted to `true`.